File: Registro Entrada Salida de Personal/Grafico.m
datos = load('BIG O.txt');

x = datos(:, 1);
y = datos(:, 2);

figure;
scatter(x, y, 'o', 'DisplayName', 'Puntos');
hold on;
plot(x, y, '-r', 'DisplayName', 'Recta que conecta los puntos');
hold off;

xlabel('# Datos');
ylabel('Tiempo');
title('Gráfico BIG O');

saveas(gcf, 'graficoMathlab.pdf');

File: Registro Entrada Salida de Personal/Grafico.R
library(ggplot2)

datos <- read.table("BIG O.txt", header = FALSE)
colnames(datos) <- c("x", "y")

grafico <- ggplot(datos, aes(x = x, y = y)) +
  geom_point() +
  geom_path() +
  labs(x = "# Daots", y = "Tiempo", title = "Gráfico BIG O")

pdf("graficoR.pdf")
print(grafico)
dev.off()
